.github/workflows/deploy_tagged.yml

```diff
@@ -84,4 +84,4 @@ jobs:
         run: echo "::set-output name=sha7::$(echo ${GITHUB_SHA} | cut -c1-7)"
 
       - name: Copy checksum files
-        run: aws s3 cp image-digest.txt  s3://${{ secrets.AWS_BUCKET }}/${{ env.maven_project_version }}-${{ steps.slug.outputs.sha7 }}/image-digest.txt
+        run: aws s3 cp image-digest.txt  s3://${{ secrets.AWS_BUCKET }}/${{ env.maven_project_version }}-${{ steps.slug.outputs.sha7 }}/dockstore-webservice/image-digest.txt
```

Comment:
Needed to update this because this is the folder dockstore-deploy is looking in when searching for the webservice image digest